server/paymentProviders/stripe/creditcard.js

```diff
@@ -161,7 +162,7 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
       stripeAccount: hostStripeAccount.username,
     });
   }
-
+  paymentIntent = handleZeroDecimalCurrencies(paymentIntent);
```

Comment:
I don't think this much, I would prefer if we create the `amount` and `application_fee_amount` straight with the right values above.

Reply:
@znarf : But notice that we are not always creating the `paymentIntent` right? On line 130; 

https://github.com/opencollective/opencollective-api/blob/e1e76bb9a46ecc4f9fab28f27c705b885337cf17/server/paymentProviders/stripe/creditcard.js#L165

it is possible that the object already contains the `paymentIntent`, in which I think we need to alter it after the creation. Am I missing something? 😄

Reply:
We are creating it in this code and not anywhere else, so it's all fine.

We don't have to handle the case of such paymentIntent already existing in our db, because we assume such case is not existing and will not exist.

Reply:
@znarf : Thanks changed the code to make sure the the amount and application_fee_amount are calculated with the right amounts. 👍🏽